test-suite/unit-tests/clib-tests/inteq.ml

```diff
@@ -0,0 +1,12 @@
+open OUnit
+open Utest
+
+let eq0 = mk_test "clib-inteq0"
+                  (TestCase (fun _ -> assert_bool "failure of Int.equal on 0" (Int.equal 0 0)))
+
+let eq42 = mk_test "clib-inteq42"
+                  (TestCase (fun _ -> assert_bool "failure of Int.equal on 42" (Int.equal 42 42)))
+
+let tests = [ eq0; eq42 ]
```

Comment:
Maybe provide a less trivial example. I am not sure we want to waste any build time testing `Int.equal`...